assume/strategies/learning_strategies.py

```diff
@@ -46,7 +49,17 @@ class RLStrategy(LearningStrategy):
     """
 
     def __init__(self, *args, **kwargs):
-        super().__init__(obs_dim=50, act_dim=2, unique_obs_dim=2, *args, **kwargs)
+        obs_dim = kwargs.get("observation_dimension", 50)
```

Comment:
the idea here was that these dimensions are fixed for this specific strategy, as they are fixed by the create_obs and create_actions functions. Defining these values from config won't work since the code has to be actively adjusted to accomodate this